### PR TITLE
Feat : Added Quiz page

### PR DESCRIPTION
**Pull request by kiranpratyush**
This is PR for the Quiz page
[Testing link](https://testing2364.netlify.app/html/quiz%20page/)

### CODE REVIEW

File: CSS/QUIZPAGE/style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 24px;
```
Comment (msdeshmukh009) [inline CSS/QUIZPAGE/style.css:8]: You can use this shorthand `padding: 24px 10px 0;` if I am not wrong.

**Review by msdeshmukh009 — APPROVED** [review]
Good to go!!

File: CSS/QUIZPAGE/style.css (hunk 1 above)
Comment (kiranpratyush) [reply CSS/QUIZPAGE/style.css]: Done.

**Review by KaustubhM1997 — APPROVED** [review]
looks good!

**Pull request merged**